tools/src/OpenSearchHttpClient.ts

```diff
@@ -22,6 +22,10 @@ export const OPENSEARCH_URL_OPTION = new Option('--opensearch-url <url>', 'URL a
   .default(DEFAULT_URL)
   .env('OPENSEARCH_URL')
 
+export const OPENSEARCH_DISTRIBUTION_OPTION = new Option('--opensearch-distribution <key>', 'OpenSearch distribution')
+  .default('opensearch.org')
+  .env('OPENSEARCH_DISTRIBUTION')
+
```

Comment:
Is there a reason this is being added into the `OpenSearchHttpClientOptions`?

Reply:
Because it will need to set the version to 2.1 for serverless because that doesn't have `GET /`.

Reply:
Moved it.